### ISSUE TITLE
VST3 more than 16 channels

### ISSUE THREAD

**Issue by ab-negative** [iPlug2/iPlug2#1133]
https://github.com/iPlug2/iPlug2/blob/b91b3abdba4639aa589d8e0baacf7e8e8cee1f6d/IPlug/VST3/IPlugVST3_ProcessorBase.cpp#L45

Does this mean VST3 support is limited to 16 channels?

**Comment by olilarkin** [comment]
No. You can implement your own function which maps channel counts to channel layouts like this

https://github.com/iPlug2/iPlug2/blob/b91b3abdba4639aa589d8e0baacf7e8e8cee1f6d/Examples/IPlugSurroundEffect/IPlugSurroundEffect.cpp#L5
